### DISCUSSION TITLE
Student Developer Pack - Bug in UX when submitting proof of scholarchip

### DISCUSSION THREAD

**Discussion by nosseb** [community/community#39095] [GitHub Education]
### Select Topic Area

Bug

### Body

Earlier today I renewed my proof of scholarship to get access to the Student Developer Pack.
For my university one has to use the laptop camera to provide a photo of an official document.
I was able to renew my access, but encountered two issues along the way (witch I suspect are linked, hens the single report) :

### 1. After taking the picture with my webcam, the OS and web browser symbol for webcam use stayed on.

After framing correctly my document in the viewer, I pressed the button to snap the picture witch had for effect to freeze the webcam viewer. So far so good.
However the hardware LED and Chrome indicator stayed activated until I closed the tab.
Since your AI did validate my scholarship, I assume it [the AI] analysed the frame that was displayed. Had it analysed a later frame, it would have seen my face rather than the document.

I'm reporting this because I believe that keeping the webcam activated for longer than necessary my be a privacy issue.
I'm also reporting this as it goes against the behaviour the user might expect, and induce him to think that the application is not responding, degrading his experience.

### 2. After pressing the green Submit button, not much happened.

Under the webcam view, there is a big green button to submit the selected picture.
Once pressed appears a text frame (orange-y, if I recall correctly) indicating that the proof of scholarship is being pre processed. This was a quick feedback, quite appreciated.

This text frame later transition to something in the like of "processing your proof, might take a many seconds" (no more orange background, different text than the previous text frame). At the same time a throbber appeared within the previously mentioned green button.
The page stayed in that state until I closed it.
The green button's throbber was still being animated.
I don't recall the web browser displaying a loading symbol.
The webcam use feedback (led & chrome icon) were still active.

While waiting I got an email from Github, welcoming me to the global campus, and after checking (in a different tab) that the student licence was active on GitHub's billing page, I chose to close the application page.

I'm using Google Chrome 107.0.5304.110 on Mac OS 13.0.1 (22A400).
I did not think to take screenshots at the time - sorry - and I can't retake the form now that I have a valid student licence.

**Comment by samus-aran** [comment]
Hi @nosseb 👋 

Thank you for such thorough feedback. We've passed this onto the team to assess. Congratulations on getting re-verified 🎉
